pkg/virt-handler/vm.go

```diff
@@ -2502,10 +2503,14 @@ func (d *VirtualMachineController) checkNetworkInterfacesForMigration(vmi *v1.Vi
 }
 
 func (d *VirtualMachineController) checkVolumesForMigration(vmi *v1.VirtualMachineInstance) (blockMigrate bool, err error) {
-
 	volumeStatusMap := make(map[string]v1.VolumeStatus)
 
 	for _, volumeStatus := range vmi.Status.VolumeStatus {
+		if volumeStatus.Name == backendstorage.PVCForVMI(vmi) && volumeStatus.PersistentVolumeClaimInfo != nil &&
+			len(volumeStatus.PersistentVolumeClaimInfo.AccessModes) == 1 &&
+			volumeStatus.PersistentVolumeClaimInfo.AccessModes[0] == k8sv1.ReadWriteOnce {
```

Comment:
Can't we use `HasSharedAccessMode`?

Reply:
Yeah I would change this condition to be more like "does not have shared access mode"